packages/front-end/pages/report/[rid].tsx

```diff
@@ -67,6 +68,13 @@ export default function ReportPage() {
         />
       )}
       <div className="mb-3">
+        {report?.experimentId && (
+          <Link href={`/experiment/${report.experimentId}`}>
+            <a>
+              <GBCircleArrowLeft /> go to experiment results
```

Comment:
I was thinking that 'back' implied you came from that page, and since ad-hoc reports could be shared or linked to directly from the profile, that may not apply. I'm not to bothered either way.